src/SpecFlow.Dnx/project.json

```diff
@@ -15,7 +14,7 @@
 	},
 
 	"frameworks": {
-		"net40": {
+		"net451": {
```

Comment:
I wasn't aware that 4.5.1 is out of support - I just updated the version to work with SpecFlow 2.

I suppose for something "temporary" and quite dirty, it doesn't matter? :confused: